bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/PulseAudioAudioSink.java

```diff
@@ -74,69 +78,123 @@ public void process(@Nullable AudioStream audioStream)
         if (audioStream == null) {
             return CompletableFuture.completedFuture(null);
         }
-        addClientCount();
-        try (ConvertedInputStream normalizedPCMStream = new ConvertedInputStream(audioStream)) {
-            for (int countAttempt = 1; countAttempt <= 2; countAttempt++) { // two attempts allowed
```

Comment:
I put this retry mechanism because I sometimes encountered "zombie connections" : They are connections that can't be seen as failed unless we try to write to them. We can encouter this especially if we use the value -1 for timeout (keeping connection).

So, I see two options : 
- keeping it
- or removing it, and also removing the ability to keep connection after playing (which is fine for me if wanted, since latency (even when reconnecting, which was the reason of this parameter) seems to have improved to absolutely not being noticeable in my last test)

Reply:
Thank you for the review @dalgwen! 

While developing I thought on the possibility of removing it (the "keep connection" functionality), but I wanted to know first whetter you found a big difference with it because in my case it's also not noticeable, thank you for testing it. I'm going to remove it then.